airbyte-webapp/src/components/ui/Card/Card.module.scss

```
@@ -35,3 +37,7 @@
     border-radius: 10px;
   }
 }
+
+.infoTooltip {
+  color: white;
```

Will be required for us to easier get the dark theme later.

```suggestion
  color: colors.$white;
```